terraform/qa-reports.tf

```diff
@@ -19,6 +19,7 @@ variable "ssh_key_path" {
 variable "ami_id" { type = "string" }
 variable "route53_zone_id" { type = "string" }
 variable "vpc_id" { type = "string" }
+variable "subnet" { type = "string" }
```

Comment:
Similar to webservers, please use values(availability_zone_to_subnet_map) instead.